rules/framework.bzl

```diff
@@ -591,6 +595,14 @@ def _bundle_dynamic_framework(ctx, is_extension_safe, avoid_deps):
         # processed infoplit its possible, but validate for the common case
         fail("Missing bundle_id: Info.plist actions require one")
 
+    # Determine the interface version of rules_apple. We don't want to force
+    # "hail mary" type rules and Bazel bumps on the community to run a given
+    # version of rules_ios and the API we depend on is relatively stable. If
+    # this ceases to be the case than consider mainlining the few components we
+    # use from it to remove this complexity.
+    rules_apple_api_version = getattr(bundling_support, "rule_api_version", None)
```

Comment:
Where does this attribute come from? I can't seem to find it in https://github.com/bazelbuild/rules_apple/blob/master/apple/internal/bundling_support.bzl or its history.

Reply:
I had missed that is was `bazel-ios` instead of `bazelbuild`. 🤦

Reply:
You've got a good point - let me update the repo to point at `bazel-ios` to make it a bit more clear.

Reply:
I've updated the README to address any concerns on this and clarify that you won't have to worry about LTS support if you're not running it:
https://github.com/bazel-ios/rules_ios/pull/655/commits/e7a635b68a046846cc26370a87c7f134d51363c5#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R182